server/api.js

```diff
@@ -74,8 +74,9 @@ router.post("/", (req, res, next) => {
 			return next(err);
 		}
 		const db = client.db("feedback-tracker");
-		const collection = db.collection("students");
+		const collection = db.collection("feedback");
 		const data = req.body;
+		console.log(data);
```

Comment:
I think it is a good practice to remove all "console.log()" before merging. As the code base becomes larger and larger it will be difficult to go back to the early stages.

Reply:
Agree.